.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: Release to npm and gh-pages.
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: A job to execute release
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install packages.
+        run: yarn install
+      - name: Set up project.
+        run: yarn build
+      - name: Run tests.
+        run: yarn test
```

Comment:
I'd consider using `test:patchset` here, becasue it runs all tests (jest too) and does linting too, and this runs on the push builds too.

Reply:
Good idea, let's do that